CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
+project(cryfs)
```

Comment:
I added the name there because Qt Creator needed it to displace the name of the project.

![snapshot](https://cloud.githubusercontent.com/assets/3496115/15736036/c3846ee8-28a5-11e6-969d-56a6acb91ea7.jpeg)